hack/update-codegen.sh

```diff
@@ -155,7 +153,8 @@ echo "CODEGEN_PKG:  $CODEGEN_PKG"
 echo "GOPATH:       $GOPATH_1"
 echo "VERIFY:       $(if is_verify_mode; then echo "true"; else echo "false"; fi)"
 echo "GO version:   $(go version)"
-
+echo "--- Environment with 'go': ---"
```

Comment:
Was this introduced only for debugging or should this stay?

Reply:
Might be helpful to know in future. But I can remove it.

Reply:
If you remove it you can also do the change proposed by @anbrsap, otherwise it is misleading.

Reply:
https://github.com/SAP/stewardci-core/pull/101/commits/7368698d9465ef9da71fd320219626f59490dae3